src/Services/Identity/Identity.API/Quickstart/Consent/ConsentController.cs

```diff
@@ -84,7 +84,7 @@ private async Task<ProcessConsentResult> ProcessConsent(ConsentInputModel model)
         var result = new ProcessConsentResult();
 
         // validate return url is still valid
-        var request = await _interaction.GetAuthorizationContextAsync(model.ReturnUrl);
+        var request = await _interaction.GetAuthorizationContextAsync(model?.ReturnUrl);
         if (request == null) return result;
 
```

Comment:
src/Services/Identity/Identity.API/Quickstart/Consent/ConsentController.cs
Duplicate of #
Duplicate of #